dotcom-rendering/src/web/components/CommentCount.importable.tsx

```diff
@@ -81,7 +81,7 @@ export const CommentCount = ({
 		joinUrl(discussionApiUrl, 'discussion', shortUrlId),
 	);
 
-	if (!commentCount && commentCount !== 0) return null;
+	if (commentCount !== undefined && commentCount !== 0) return null;
```

Comment:
I do not understand this logic… should it not be checking whether there are at least one comment?

```suggestion
	if (commentCount === undefined || commentCount < 1) return null;
```